h3i/src/prompts/h3/mod.rs

```diff
@@ -473,6 +479,30 @@ pub fn prompt_stream_bytes() -> InquireResult<Action> {
     })
 }
 
+pub fn prompt_send_datagram(with_quarter_stream: bool) -> InquireResult<Action> {
+    if with_quarter_stream {
+        let stream_id = h3::prompt_stream_id()?;
+        // https://www.rfc-editor.org/rfc/rfc9297#name-http-3-datagrams
+        let quarter_stream_id = stream_id >> 2;
+
+        let payload = Text::new("payload bytes:").prompt()?;
+
+        let len = octets::varint_len(quarter_stream_id) + payload.len();
+        let mut d = vec![0; len];
+        let mut b = octets::OctetsMut::with_slice(&mut d);
+        b.put_varint(quarter_stream_id)
+            .map_err(|e| InquireError::Custom(Box::new(e)))?;
```

Comment:
Wasn't sure if this was any better/worse than just `unwrap()`ing here 🤷

Reply:
yeah you can just `unwrap()` here, that's fine